docs/roadmap.md

```diff
@@ -1,38 +1,98 @@
 # Roadmap
 
-The aim is to deliver a Minimum Viable Product (MVP). In a nutshell, an MVP
-seeks to collect the maximum amount of validated learning about our users in a
-short time.
+## 2022
 
-Based on the user feedback we collect, we develop features that add up towards
-the goal of delivering the MVP. We're working on phases:
+### Quarter 1
 
-## Phase 1
+- Brainstorm process
 
-- Use SQLite as storage backend.
-- Running the s3gw in a container.
-- Be able to run basic S3 operations.
-- Ability to deploy the S3 Gateway on a K8s or K3s cluster with.
-- [Design mockups][1] for User management & an S3 explorer.
+---
+
+### Quarter 2
+
+- Initial project setup
+- First tests with RGW
+- [UI mockups][1]
+
+---
+
+### Quarter 3
+
+- File-based backend
+- Basic S3 operations support
+- User management
+- Helm charts
+- Testing
+- UI implementation
+- Documentation
+- Basic object management & versioning
+- Bucket management
+- Automation implementation
+
+---
+
+### Quarter 4
+
+- Object deletion/undeletion
+- Multipart uploads
+- Multipart copy
+- ACL support
+- UI: Basic object explorer
+- Rancher Partner chart
+
+## 2023
+
+<!-- markdownlint-disable MD024 -->
+
+### Quarter 1
+
+- [Expiration Lifecycle Management (unversioned)][2]
+- [Object locks][3]
+- [Object listing with prefixes/filters][4]
+- [Telemetry][5]
+- [CRD based configuration of buckets and rbac][6]
+- [UI: Object deletion/undeletion][7]
 
 ---
 
-## Phase 2
+### Quarter 2
 
-- Basic S3 explorer UI
-- Helm chart
-- Define & start work on a file-based backend
-- Implement CI/CD
+- [Expiration Lifecycle Management (versioned)][8]
+- [Access & Identity Management][10]
+- [Metrics from backend][9]
+- [UI: Object versioning][11]
 
 ---
 
-## Phase 3
+### Quarter 3
 
-- File-based backend integration
-- Increased support for S3 functional features
-- S3 explorer UI
-- User management UI
+- [Implement Amazon Security Token][12]
+- [Cross-site replication][13]
```

Comment:
I still think this is a bad idea, having it on the roadmap for next year. This should be a "we'll work on it should there be real need, but otherwise it will be pushed further into the future" kind of thing.

Reply:
Keep in mind that the roadmap is the closest we have to an official commitment of intent.

Once it goes on the roadmap we are sending the message this is something we consider to be in scope, regardless of whether it's in Q4 or Q _N_ 202X. And I don't think we want to consider this in scope for now. Hence why it should not be in the roadmap, but which we may still keep somewhere as a "nice to have, but we're not working on it" kind of thing.

Reply:
Acknowledged. This is something we'd discussed in the past but isn't reflected here. Let me redo the list a bit

Reply:
Ok, I restructured it, using @l-mb 's suggestion. I think it is best to avoid creating false expectations. I've also improved the disclaimer stating that this roadmap is subject to change.